src/Providers/Stripe/Service.php

```diff
@@ -77,4 +100,74 @@ public function update(string $uid): Payment
 
         return new Payment($response->id, $response->currency, $response->amount, $response->created);
     }
+
+    /**
+     * Creates Stripe customer.
+     *
+     * @param string $email
+     * @param string $paymentMethod
+     * @param string|null $description
+     *
+     * @return Customer
+     */
+    public function createCustomer(string $email, string $paymentMethod, ?string $description = null): Customer
```

Comment:
I think I agree with @adteulade that this belongs in another service class

Reply:
again I feel description isn't necessary / qualifies as meta-data, what is this used for currently?